kernel/src/listed_log_files.rs

```diff
@@ -95,7 +93,7 @@ fn list_log_files(
                 // normal `version.crc` files which are listed + captured normally. Additionally
                 // we likely aren't even 'seeing' these files since lexicographically the string
                 // "." comes before the string "0".
-                .filter_map_ok(identity)
+                .filter_map_ok(|path_opt| path_opt.filter(|p| p.should_list()))
```

Comment:
hrmm, I don't know if this is the right place for this. Like maybe I _do_ want to know which staged commits exist (because I want to backfill say). Perhaps this should be filtered out at a higher level that knows semantically it doesn't want those files?